credoai/evaluators/utils/utils.py

```diff
@@ -0,0 +1,67 @@
+import inspect
+import sys
+
+
+def list_assessments_exhaustive():
+    """List all defined assessments"""
+    return inspect.getmembers(
+        sys.modules[__name__],
+        lambda member: inspect.isclass(member) and member.__module__ == __name__,
+    )
+
+
+def list_assessments():
+    """List subset of all defined assessments where the module is importable"""
+    assessments = list_assessments_exhaustive()
+    usable_assessments = []
+    for assessment in assessments:
+        try:
+            _ = assessment[1]()
+            usable_assessments.append(assessment)
+        except AttributeError:
+            pass
+    return usable_assessments
+
+
+def get_assessment_names():
+    return {name: assessment_class().name for name, assessment_class in ASSESSMENTS}
```

Comment:
`ASSESSMENTS` seems to be missing. In general, is the code here used anywhere atm, or is this more like a placeholder?